src/components/toast/index.js

```diff
@@ -195,6 +196,7 @@ export default class Toast extends BaseComponent {
         ]}
         {...animationConfig}
       >
+        {blur && <BlurView style={this.styles.blur} blurType={blur.type || 'light'} blurAmount={blur.amount || 3}/>}
```

Comment:
maybe create helper function to get blurOptions (handling defaults) and spread it here